app/tests/spec/lib/url.js

```diff
@@ -142,6 +142,17 @@ function (chai, _, Url) {
         assert.equal(Url.removeParamFromSearchString('foo', ''), '');
       });
     });
+
+    describe('isNavigable', function () {
+      it('detects HTTP and HTTPS properly', function () {
+        assert.isFalse(Url.isNavigable('urn:ietf:wg:oauth:2.0:fx:webchannel'));
+        assert.isTrue(Url.isNavigable('https://find.firefox.com'));
+        assert.isTrue(Url.isNavigable('http://find.firefox.com'));
```

Comment:
Can you add a test for `telnet://` or `ftp://` or something with a protocol:// that is not http/https?